profile.graphqls

```diff
@@ -156,6 +163,10 @@ extend type Query {
     getProfileTaskSegmentList(taskID: String): [BasicTrace!]!
     # query profiled segment
     getProfiledSegment(segmentId: String): ProfiledSegment
+    # query combined segments by ref
+    getProfiledSegments(segmentIds: [String!]!): [ProfiledSegments!]!
```

Comment:
What do you mean `by ref`?  It is better query by Trace ID?

Reply:
This means combining all the same segments under one process. 
Query by segments because this query only focuses on the profiled segments, not all segments under one trace.

Reply:
Are you going to verify the segment data somehow?

Reply:
No verification, just query the segments and combine them when the cross-thread segment matches with the parent segment. 
Then, The UI can show the spans in the same process, and analyze the profiling results with the same process(segment id list).

Reply:
And why do we need `ProfiledSegments` Array and this new object rather than only `ProfiledSegment` array only?

Reply:
> No verification, just query the segments and combine them when the cross-thread segment matches with the parent segment. Then, The UI can show the spans in the same process, and analyze the profiling results with the same process(segment id list).

I think UI is hard to verify things like `ref` linking segments. We may only bind segments with trace ID belonging to one task ID. This should be enough.

Reply:
Because they may become different traces/processes, this decide by the query.

Reply:
> > No verification, just query the segments and combine them when the cross-thread segment matches with the parent segment. Then, The UI can show the spans in the same process, and analyze the profiling results with the same process(segment id list).
> 
> I think UI is hard to verify things like `ref` linking segments. We may only bind segments with trace ID belonging to one task ID. This should be enough.

Do you mean to query all segments in the same trace? But there may have multiple segments that are not profiled, so they cannot be analyzed.